elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -28,17 +33,36 @@ def categorize_options(element):
             else:
                 incorrect_answers.append(answer_tuple)
             index += 1
+
+    file_path = pl.get_string_attrib(element, 'external-json', EXTERNAL_JSON_DEFAULT)
+    if file_path is not EXTERNAL_JSON_DEFAULT:
+        correct_attrib = pl.get_string_attrib(element, 'external-json-correct', EXTERNAL_JSON_DEFAULT_CORRECT)
+        incorrect_attrib = pl.get_string_attrib(element, 'external-json-incorrect', EXTERNAL_JSON_DEFAULT_INCORRECT)
+        if pathlib.PurePath(file_path).is_absolute():
+            json_file = file_path
+        else:
+            json_file = pathlib.PurePath(data['options']['question_path']).joinpath(file_path)
+        with open(json_file, mode='r', encoding='utf-8') as f:
+            obj = json.load(f)
```

Comment:
Maybe lets be very explicit here and throw our own error if the file is not found.  Something like `external JSON answer file could not be found` so that the user knows exactly what went wrong, instead of thinking that something broke internally.

Reply:
Implemented 👍

Reply:
I also display the resolved file path to make it easier for user to debug 👍